lib/lib.es2015.iterable.d.ts

```diff
@@ -223,15 +223,15 @@ interface PromiseConstructor {
      * @param values An array of Promises.
      * @returns A new Promise.
      */
-    all<TAll>(values: Iterable<TAll | PromiseLike<TAll>>): Promise<TAll[]>;
+    all<TAll>(values: Iterable<TAll>): Promise<(awaited TAll)[]>;
```

Comment:
I recommend changing `<TAll>` to `<T>`:
```suggestion
    all<T>(values: Iterable<T>): Promise<(awaited T)[]>;
```